src/static/css/infusion-docs.css

```diff
@@ -69,7 +69,7 @@ td {
 .fl-prefsEditor-buttons button:focus,
 .infusion-docs-TOC li a:focus .infusion-docs-TOC-pageName,
 a:focus {
-    outline: solid 0.2rem #008cba;
+    outline: solid 0.2rem hsl(195, 100%, 31%);
```

Comment:
Out or curiosity, why the switch to hsl? Also I'd assume you'd typically do these types of changes with CSS custom properties. Are you thinking of leaving that for a larger reworking of the entire stylesheet?

Reply:
Switch to HSL was so that I could darken the current hue without changing its other properties (it was `hsl(195, 100%, 35%)` prior to this change). And yes, I don't think making any structural changes to the stylesheet makes sense at this point— there's a lot to consider.